Makefile

```
@@ -229,9 +234,15 @@ copyright:
 	@printf $(COLOR) "Fix license header..."
 	@go run ./cmd/tools/copyright/licensegen.go
 
+goimports:
```

This should depend on the modified files: e.g.
```suggestion
goimports: $(MODIFIED_FILES)
```

---

Err, I guess since this rule doesn't have any file output, then this wouldn't help with caching actually